source/_integrations/tank_utility.markdown

```diff
@@ -75,8 +75,8 @@ password:
   description: "Your [https://app.tankutility.com](https://app.tankutility.com) password."
   required: true
   type: string
-unit_of_measurement:
-  description: All devices to monitor.
+devices:
+  description: All devices to monitor.  Double quotes have been reported as required around each device string to configure the integration.
```

Comment:
This is not a device/integration issue. This is about YAML. I don't think we should describe how YAML works hidden in a note at a configuration option.

Reply:
OK fair enough.  Will edit out that sentence, other changes remain.